release/create_release_archives.sh

```diff
@@ -115,6 +115,7 @@ find samples install -type f \( \
   -o -name "*.md" \
   -o -name "kubeconfig" \
   -o -name "*.jinja*" \
+  -o -name "*.sh" \
```

Comment:
hmm, this includes the following scripts.

```bash
install/updateVersion.sh
install/kubernetes/webhook-patch-ca-bundle.sh
install/kubernetes/webhook-create-signed-cert.sh
install/tools/setupMeshEx.sh
install/tools/setupIstioVM.sh
samples/bookinfo/consul/cleanup.sh
samples/bookinfo/build_push_update_images.sh
samples/bookinfo/src/build-services.sh
samples/bookinfo/src/mongodb/script.sh
samples/bookinfo/kube/cleanup.sh
samples/bookinfo/eureka/cleanup.sh
samples/rawvm/demo.sh
samples/helloworld/src/build_service.sh
samples/helloworld/loadgen.sh
```

Reply:
it's ok I think, can you | wc but I expect this to add a couple kbytes
the install/tools/ are already added anyway - I wonder which part of the script (that was specifically adding install/tools/*) can be removed if we have this one

Reply:
```bash
$ find install/ samples/ -name "*.sh" | xargs wc
  362  1294 15981 install/updateVersion.sh
   15    41   360 install/kubernetes/webhook-patch-ca-bundle.sh
  119   385  3142 install/kubernetes/webhook-create-signed-cert.sh
  216  1141  8407 install/tools/setupMeshEx.sh
  113   464  3597 install/tools/setupIstioVM.sh
   58   228  1541 samples/bookinfo/consul/cleanup.sh
   30   140  1058 samples/bookinfo/build_push_update_images.sh
   66   271  2386 samples/bookinfo/src/build-services.sh
    3    14   121 samples/bookinfo/src/mongodb/script.sh
   57   221  1476 samples/bookinfo/kube/cleanup.sh
   58   228  1541 samples/bookinfo/eureka/cleanup.sh
   45   246  1983 samples/rawvm/demo.sh
   22   123   865 samples/helloworld/src/build_service.sh
   17   106   681 samples/helloworld/loadgen.sh
 1181  4902 43139 total
```

Reply:
sgtm, 12k/43k are already in the release

Reply:
Instead of "*.sh" would "webhook*.sh" (or "*/*/webhook*.sh" [I'm not all that familiar with the syntax for find]) be sufficient?

It's not the space I'm concerned about, but really about 1) confusing new users with stuff they don't need (or might feel like they need to try to understand), and 2) a dev being afraid to remove the extra files in the future if they don't know for sure if somebody needs them (or why).

As for install/tools, it's handled just a few lines down by this (which offhand seems like a simple 'cp' [or mkdir & cp] could have been sufficient):

find install/tools -type f -exec ${CP} --parents {} "${COMMON_FILES_DIR}" \;

Reply:
Agree on both points - we will need to sanitize the release, in particular the install part.

For samples - it's probably ok to be more relaxed, but I'm not sure how much they are tested or expected to work, 
may be better to eventually move them to the 'contrib' repo and just point users to it.

Reply:
Updated the PR to explicitly copy the two webhook scripts from the existing location. These files are k8s specific so it makes some sense to keep them under a k8s specific directly. They're also already referenced by istio.io docs. Sounds like there is a larger issue of sanitizing / organizing our releases that is warranted? Is there a tracking issue?